string-manipulation-2/src/last-chars.js

```diff
@@ -0,0 +1,19 @@
+/* exported lastChars */
+
+// make a new variable with empty string
+// if the length is less than string,
+// loop value from string.length - length and decrement string.length - length
+// if the length is larger than string, return string
+// output new variable
+
+function lastChars(length, string) {
+  var newString = '';
+  for (var i = string.length - length; i < string.length; i++) {
+    if (length < string.length) {
```

Comment:
You should check for this once before the loop so that you don't need to do this conditional on each iteration.